db/migrate/20230228212151_add_roles_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddRolesToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :admin_role, :boolean, default: false
+    add_column :users, :user_role, :boolean, default: true
```

Comment:
It looks weird. First of all, I don't see any updates of the file `db/schema.rb` in the current PR. And, secondly, there is already a file `role` for `users` table, so why do we need new ones?

Reply:
you are right, forget to delete this migration, not used in the final solution.